plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaEventListenerConfig.java

```diff
@@ -43,6 +43,8 @@ public class KafkaEventListenerConfig
     private boolean publishCreatedEvent = true;
     private boolean publishCompletedEvent = true;
     private boolean publishSplitCompletedEvent;
+    private Long maxRequestSize = 5242880L;
+    private Long batchSize = 16384L;
```

Comment:
Could you leave a code comment explaining how these values are chosen? 

Please use `long` instead.